src/components/BareMetalHosts/StatusComponents.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Popover } from '@patternfly/react-core';
+import { Icon } from 'patternfly-react';
+
+import { getHostStatusText } from '../../selectors';
+
+export const IconAndText = ({ icon, text }) => (
+  <React.Fragment>
+    <Icon type="pf" name={icon} className="co-icon-and-text__icon" />
```

Comment:
please rename this class to `kubevirt-icon-and-text` and define required margin there